pkg/smbios/smbios.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2016-2021 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package smbios
+
+const (
+	SMBIOS2HeaderSize = 0x1f
```

Comment:
unexport these constants, since they're only used in implementation inside package

Reply:
rename it to smbios2HeaderSize